src/publish.builds

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$(ToolsDir)PublishContent.targets" />
+  <Import Project="$(ToolsDir)versioning.targets" />
+
+  <PropertyGroup>
+    <PublishPattern>$(PackageOutputRoot)**\*.nupkg</PublishPattern>
+  </PropertyGroup>
+
+  <Target Name="CreateContainerName" DependsOnTargets="CreateVersionFileDuringBuild">
+    <PropertyGroup>
+      <ContainerName>corefx-rc3-$(BuildNumberMajor)-$(BuildNumberMinor)</ContainerName>
```

Comment:
Is this PreReleaseLabel in Packaging.targets?  If so I will use this property (I had hard-coded this to unblock the test build and forgot about it).